docs/01_cva6_user/ip-xact/cva6_csr.xml

```diff
@@ -2004,14 +2004,14 @@ with the index of the bit position equal to the value returned in the ``mcause``
                                 <ipxact:register>
                                         <ipxact:name>mscratch</ipxact:name>
                                         <ipxact:displayName>Machine Scratch</ipxact:displayName>
-                                        <ipxact:description>This register is used to hold a pointer to a machine-mode hart-local context space and swapped with a user register upon entry to an M-mode trap handler.</ipxact:description>
+                                        <ipxact:description>This register is used to hold a value dedicated to Machine mode. Attempts to access without Machine mode level raise illegal instruction exception.</ipxact:description>

```

Comment:
I would suggest to not mention access mode rights in the description (to avoid redundant information as it can be derived from the register address offset MSB).

Reply:
As agreed with @frikhaAziz we will rely on ip-xact information to specify the CSR behaviour in md and rst output files.